backend/src/controllers/users/users.controller.ts

```diff
@@ -22,15 +33,16 @@ const getUsers = async (
 };
 
 const updateUser = async (
-  req: UserUpdateInput,
+  req: TypedRequest<{ id: string }, UserUpdateInput>,
   res: Response,
   next: NextFunction,
 ): Promise<void> => {
   try {
     const { id: userId } = req.params;
-    const userData = req.body;
+    const { tokenPayload, ...userData } = req.body;
     const user = await usersService.updateUser(userId, userData);
 
+    console.log(tokenPayload);
```

Comment:
I think this is not the best way to get rid of the `tokenPayload` here and pass new object further. But for now, this is what I did. I will be grateful for advice.

Reply:
No, it's valid way to get stip off some properties from an object